app/assets/stylesheets/components/gitter_floating.scss

```diff
@@ -1,21 +1,53 @@
-.chat-floating-btn {
-  background: $button-primary;
-  border-radius: 50%;
+.floating-dropdown {
   bottom: 20px;
-  box-shadow: 0 0 15px 2px rgba(0,0,0,0.2);
-  cursor: pointer;
-  padding: 1em;
   position: fixed;
-  right: 20px;
   z-index: 1000;
+  right: 20px;
+
+  .floating-dropdown-item {
+    background: $button-primary;
+    border-radius: 50%;
+    box-shadow: 0 0 15px 2px rgba(0,0,0,0.2);
+    padding: 1em;
+    cursor: pointer;
+
+    &:hover {
+      box-shadow: 0 0 15px 6px rgba(0,0,0,0.2);
+    }
 
-  &:hover {
-    box-shadow: 0 0 15px 6px rgba(0,0,0,0.2);
+    &.main-floating-btn {
+      position: relative;
+      z-index: 2;
+
+      .fa.fa-comments {
+        color: $white;
+        font-size: 2em;
+      }
+    }
+
+    &.chat-floating-btn, &.forum-floating-btn {
+      margin: 0 0 -50px 0;
+      opacity: 0;
+      transition: margin 1s ease-out, opacity 0.5s;
+      padding: 0.5em;
+      margin-right: 0.6em;
+      margin-left: 0.6em;
+      z-index: 1;
+      position: relative;
+
+      .fa {
+        color: $white;
+        font-size: 1.2em;
+      }
+    }
   }
+}
 
-  .fa.fa-comments {
-    font-size: 2em;
-    color: $white;
+.floating-dropdown:hover {
```

Comment:
Can we move this hover under the `.floating-dropdown ` scope above please?

Reply:
This did not behave the same for some reason.